taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -126,10 +141,36 @@ const AccumulateAllComponents = ({ id, taxonomyName, branchName }) => {
                 nodeObject={nodeObject}
                 setNodeObject={setNodeObject}
               />
+              {/* Fab for submitting edits */}
               <ListAllEntryProperties
                 nodeObject={nodeObject}
                 setNodeObject={setNodeObject}
               />
+              {changesMade && (
+                <div
+                  style={{
+                    backgroundColor: "white",
+                    position: "sticky",
+                    bottom: 0,
+                    zIndex: 1,
+                  }}
+                >
+                  <Button
+                    onClick={handleSubmit}
+                    variant="contained"
+                    sx={{
+                      minHeight : "50px",
+                      borderRadius : "20px",
```

Comment:
If possible, please make the dimensions in relative units. Will be easier when everything is being made responsive.